recipe/meta.yaml

```
@@ -51,6 +51,7 @@ outputs:
       build:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
+        - {{ stdlib("c") }}
```

As a workaround let's try adding a selector to limit `{{ stdlib("c") }}` to Linux. This shouldn't be needed as the `cuda-nvtx` package is Linux only. Though maybe this helps `conda-build` in its parsing of this recipe

```suggestion
        - {{ stdlib("c") }}  # [linux]
```

---

Looks like this is still needed even with `conda-build` version `24.3.0`

```suggestion
        - {{ stdlib("c") }}  # [linux]
```